code/modules/admin/verbs/maprotation.dm

```diff
@@ -68,41 +65,71 @@
 			return
 
 		qdel(M)
+		var/config_file = null
+		var/list/json_value = list()
+		var/config = tgui_alert(usr,"Would you like to upload an additional config for this map?", "Map Config", list("Yes", "No"))
+		if(config == "Yes")
+			config_file = input("Pick file:", "Config JSON File") as null|file
+			if(isnull(config_file))
+				return
+			if(copytext("[config_file]", -5) != ".json")
+				to_chat(src, span_warning("Filename must end in '.json': [config_file]"))
+				return
+			if(fexists("data/custom_map_json/[config_file]"))
+				fdel("data/custom_map_json/[config_file]")
+			if(!fcopy(config_file, "data/custom_map_json/[config_file]"))
+				return
+			if (virtual_map.LoadConfig("data/custom_map_json/[config_file]", TRUE) != TRUE)
+				to_chat(src, span_warning("Failed to load config: [config_file]. Check that the fields are filled out correctly. \"map_path\": \"custom\" and \"map_file\": \"your_map_name.dmm\""))
```

Comment:
Damn, I changed the configuration path on the recommendation and completely forgot about it.